app/components/public/session-item.js

```diff
@@ -15,4 +16,9 @@ export default class SessionItem extends Component {
       this.set('hideImage', false);
     }
   }
+
+  @computed('session.event', 'session.startsAt')
+  get sessionStartsAtInEventTimezone() {
+    return moment.tz(this.session.startsAt, this.session.event.get('timezone'));
```

Comment:
What happens if you use `session.event.timezone`?

Reply:
It tells that since `session.event` is a proxy, it will be better that I use `.get()` rather than directly accessing that property.

Reply:
Then wrap event in EmberObject in the test

Reply:
Like this ?
```js
`event: EmberObject.create({ timezone: 'Asia/Kolkata' })
```

Reply:
Yes